splunkapp/README.md

```diff
@@ -0,0 +1,67 @@
+Wazuh
+=====
+
+[![Slack](https://img.shields.io/badge/slack-join-blue.svg)](https://wazuh.com/community/join-us-on-slack/)
+[![Email](https://img.shields.io/badge/email-join-blue.svg)](https://groups.google.com/forum/#!forum/wazuh)
+[![Documentation](https://img.shields.io/badge/docs-view-green.svg)](https://documentation.wazuh.com)
+[![Documentation](https://img.shields.io/badge/web-view-green.svg)](https://wazuh.com)
+
+Wazuh is an Open Source Host-based Intrusion Detection System that performs log analysis, file integrity monitoring, policy monitoring, rootkit detection, real-time alerting, active response, vulnerability detector, etc.
+
+In this repository, you can find the necessary tools to build a Wazuh Splunk app package.
+
+## Tools needed to build the package
+
+To build a Wazuh Splunk app package you need to install the following tools:
+  - `docker`: [installation guide](https://docs.docker.com/install/linux/docker-ce/centos/) for RPM and [installation guide](https://docs.docker.com/install/linux/docker-ce/debian/) for Debian.
+- `git`:  [installation guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). 
+  
+## Building Wazuh Splunk app packages
+
+To build a Splunk app package, you need to download this repository and use the `generate_wazuh_splunk_app.sh` script. This script will download the source code from the [wazuh/wazuh-splunk](https://github.com/wazuh/wazuh-splunk) repository build a Docker image with all the necessary tools to build the app package and run a Docker container from that image that will generate a `.tar.gz` package.
```

Comment:
Please review if a "," is necessary between "will download the source code from the repository build a Docker. Review all places with this sentence.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/162/commits/24140971527228f1b10c43b13ac492b6e8409fda